pkg/defaults/defaults.go

```diff
@@ -62,10 +62,12 @@ const (
 	DefaultEveRegistry          = "lfedge"
 	DefaultRegistry             = "docker.io"
 
-	DefaultSFTPUser     = "user"
-	DefaultSFTPPassword = "password"
+	DefaultSFTPUser      = "user"
```

Comment:
let's use something harder as a default. Many bruteforcers are guessing that combo. Also it's a rule of thumb that if you have the default then 95% of users will use that default.